romana-install/roles/romana/kubernetes-postinstall/files/demo/demo.sh

```diff
@@ -21,7 +21,7 @@ function maybe_first_prompt() {
 }
 function run() {
     maybe_first_prompt
-    rate=25
+    rate=$((25 * (1 + ${#1} / 30)))
```

Comment:
What are we doing here?

I'm sure it's nice and all... whatever it is. Just curious.